src/ReactLeafletMap/layers/CanvasVectorLayer/index.js

```diff
@@ -1,33 +1,42 @@
-import {MapLayer, withLeaflet} from 'react-leaflet';
+import React from 'react';
+import {createLayerComponent} from '@react-leaflet/core';
 import LeafletCanvasLayer from './LeafletCanvasLayer';
+import paneHelpers from '../../helpers/pane';
 
-class CanvasVectorLayer extends MapLayer {
-	createLeafletElement(props) {
-		let layer = new LeafletCanvasLayer({
-			paneName: props.uniqueLayerKey,
-			paneZindex: props.zIndex,
-		});
-		layer.setProps(props);
-		return layer;
-	}
+function createLeafletElement(props, context) {
+	const paneKey = paneHelpers.getKey(props.mapKey, props.uniqueLayerKey);
+	const instance = new LeafletCanvasLayer({
+		paneName: paneKey,
+		paneZindex: props.zIndex,
+	});
 
-	updateLeafletElement(fromProps, toProps) {
-		super.updateLeafletElement(fromProps, toProps);
+	setTimeout(() => {
```

Comment:
@vlach1989 is it really necessary? Add some comment why timeout is needed.

Reply:
Yes, it is: 5f3dd6ebe664feb0c66d696237434d82ca8293dd